print-bgp.c

```diff
@@ -2555,6 +2609,23 @@ bgp_capabilities_print(netdissect_options *ndo,
                tok2str(bgp_safi_values, "Unknown", GET_U_1(opt + i + 5)),
                GET_U_1(opt + i + 5));
             break;
+        case BGP_CAPCODE_BGPSEC:
+            /* Version (4 bits), Direction (1 bit), Flags (3 bits), AFI (16 bits) */
+            cap_offset = 1;
+            if (cap_len < 3) {
+                ND_PRINT(" (too short, < 3)");
+                return;
+            }
+
+            ND_PRINT("\n\t\tVersion %u, Direction %s (%u), Flags 0x%01x, AFI %s (%u)",
+                      GET_U_1(opt + i + 2)&0xf0,
+                      (GET_U_1(opt + i + 2)&0x08) ? "Send" : "Receive",
+                      (GET_U_1(opt + i + 2)&0x08)>>3,
+                      GET_U_1(opt + i + 2)&0x07,
```

Comment:
Should this use `bittok2str()` or `bittok2str_nosep()` to show the individual flags, rather than just dumping the flag value as a decimal number?

Reply:
Or should it just ignore those bits, as [RFC 8205 section 2.1](https://datatracker.ietf.org/doc/html/rfc8205#section-2.1) appears to show them as "Unassigned"?

Reply:
Right, `Flags` are currently ignored (and are thus always 0). If it is common in tcpdump to omit such fields, I'm all for it to scratch them.